tsconfig.json

```diff
@@ -46,7 +46,9 @@
 			"@umbraco-cms/backoffice/collection": ["libs/collection"],
 			"@umbraco-cms/backoffice/tree": ["libs/tree"],
 			"@umbraco-cms/backoffice/section": ["libs/section"],
+			"@umbraco-cms/backoffice/variant": ["libs/variant"],
 
+			"@umbraco-cms/backoffice/core/components": ["src/backoffice/core/components"],
```

Comment:
I like this change, just so you know, the rollup setup for libraries do not support multiple entrypoints at the moment, e.g. `/core/components`, because it only builds against the root index.ts file in each library. So we probably need to do something either with the paths or with the rollup setup later on when we get an actual "core" library.

Reply:
I don't think it breaks anything. Merely just a heads up if you wanted to move the "core" folder into our libs folder and expect it to work out of the box :)